internal/gatewayapi/testdata/httproute-with-redirect-filter-full-path-replace-https.in.yaml

```diff
@@ -31,9 +31,6 @@ httpRoutes:
     - matches:
       - path:
           value: "/"
-      backendRefs:
```

Comment:
were these interfering with the tests ?

Reply:
nope, sorry. i've repushed this without any extra changes to keep it to a pure single-purpose fix